src/main/java/br/com/sbk/sbking/networking/server/gameServer/MinibridgeGameServer.java

```diff
@@ -31,6 +31,12 @@ public void run() {
     LOGGER.info("Sleeping for 500ms waiting for clients to setup themselves");
     sleepFor(500);
 
+    LOGGER.info("Sleeping while nobody is connected");
+    while (this.sbkingServer.nobodyIsConnected()) {
```

Comment:
is this loop necessary on other game servers as well, such as the KingGameServer?

Maybe all servers should extend some basicGameServer with behaviors such as this busy wait if so

Reply:
For sure. This will be implemented as part of the Lobby feature #17 when the `GameServer`s will be refactored to be really nice and polymorphic!